lib/node.js

```diff
@@ -38,11 +47,7 @@ const load = async (fileName, importObject = {}) => {
     }
     exports[name] = (...args) => {
       if (typeof args.at(-1) !== 'function') return fn(...args);
-      let callbacks = calls.get(name);
-      if (!callbacks) {
-        callbacks = [];
-        calls.set(name, callbacks);
-      }
+      const callbacks = callbacksRegistry.getOrSetDefault(name, []);
```

Comment:
- Do not pass empty array to this method
- Rename `.getOrSetDefault(name)` to `.getCallbacks(name)`